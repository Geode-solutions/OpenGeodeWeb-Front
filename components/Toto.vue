<template>
  <h1>Coucou</h1>
</template>
