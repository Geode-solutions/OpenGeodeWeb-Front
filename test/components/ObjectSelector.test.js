/**
 * @vitest-environment happy-dom
 */
import { describe, expect, it, vi } from "vitest"
import { mount } from "@vue/test-utils"
import Toto from "../../components/Toto.vue"

describe("ErrorBanner", () => {
  it("Renders properly", () => {
    const wrapper = mount(Toto)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
