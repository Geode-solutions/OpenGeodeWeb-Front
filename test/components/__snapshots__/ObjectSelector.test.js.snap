// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ErrorBanner > Renders properly 1`] = `"<h1>Coucou</h1>"`;
